Package.swift

```diff
@@ -10,14 +10,14 @@ let package = Package(
         .library(
```

Comment:
Any reason for this particular change here? If anything, it should be consistent in both cases (i.e. no comma or both lines should have had a comma).

Reply:
This change was actually introduced as part of the [previous PR](https://github.com/fingerprintjs/fingerprintjs-ios/pull/13), so I am surprised that you haven't asked this question there. Anyway, what is the reason for this change? The reason is basically the `swift-format` that was introduced with [this commit](https://github.com/fingerprintjs/fingerprintjs-ios/commit/c7a87916f63f46fa3e26e1ce4010d7c4b44142f0) back in March. It's just that `Package.swift` was not taken into account when linting Swift files, but the previous PR fixed that. These are the warnings that were reported by `swift-format`:
<img width="869" alt="Screenshot 2022-11-29 at 16 39 29" src="https://user-images.githubusercontent.com/10102561/204574371-262b4a69-e877-4d7e-861c-38cb458b0bfd.png">
If this kind of formatting is not what we want, we should take a look at `.swift-format` configuration file and see if we can customize that `TrailingComma` rule to our needs.

Reply:
"`Package.swift` wasn't part of the formatting rules" would have been sufficient explanation.